snap/snapcraft.yaml

```diff
@@ -36,6 +36,13 @@ parts:
 
       # Don't overly optimize for build CPU-- stay generic
       - -DBINARY_PACKAGE_BUILD=On
+
+    organize:
+      # Darktable uses a relative path to find the lensfun database, which is
```

Comment:
Can you show me where the darktable source does this? It's not just the datadir?

Reply:
The latter. It seems to look for the lensfun db next to the datadir, and `usr/share` is where the lensfun package installs that data.